### PR TITLE
Pin tooling versions

### PR DESCRIPTION
**Pull request by lawrencegripper**
This PR pins the versions of tooling used in the devcontainer to ensure future builds of the container are reproducable. 

The driver for this change was issues observed with `gopls` and `kubectl` in another project. 

For `kubectl` v1.17 change validation behavior which introduced a breaking change when using some scripts. 

For `gopls` the location of the files in the repo moved which caused the devcontainer to stop building correctly. 

I've raised this as draft as it would be good to get #152 in first so the devcontainer changes are built before being merged.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -57,12 +31,32 @@ RUN apt-get update \
     && apt-get update \
     && apt-get install -y docker-ce-cli \
     # Install kubectl
-    && curl -sSL -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+    && curl -sSL -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$KubectlVersion/bin/linux/amd64/kubectl \
     && chmod +x /usr/local/bin/kubectl \
     # Install Helm
-    && curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash \
+    && DESIRED_VERSION=$HelmVersion curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash \
```
Comment (lawrencegripper) [inline .devcontainer/Dockerfile]: Todo: not quite working right - fix on the way.

Comment (lawrencegripper) [reply .devcontainer/Dockerfile]: Sorted.

**Comment by stuartleeks** [comment]
@lawrencegripper - now that #152 is merged, are you ok to publish this?

**Comment by lawrencegripper** [comment]
Will do, let me rebase now and push

**Comment by stuartleeks** [comment]
Thanks @lawrencegripper  - are you ok to mark as Ready for Review now?

**Comment by lawrencegripper** [comment]
Just testing some stuff out atm, `make set-kindcluster` seems to have an issue so will fix up then push. Sorry for the delay

**Comment by stuartleeks** [comment]
np - thanks for updating. (was just checking in case you'd made the update but not marked as ready for review.... not that I'd ever do something like that 😉)

**Comment by lawrencegripper** [comment]
Was an issue my side, good to go with review.

**Comment by lawrencegripper** [comment]
@Azadehkhojandi / @JasonTheDeveloper are you to have a review of this one, @EliiseS is working on some changes to the devcontainer for the load testing work and would be good to base them off this one.

**Pull request merged**